<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginWindow</class>
 <widget class="QMainWindow" name="loginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>loginWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #f0f0f0;  /* 淺灰色背景 */
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>210</y>
      <width>541</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="Account">
       <property name="styleSheet">
        <string notr="true">QLabel {
    font: 13pt &quot;Microsoft YaHei UI&quot;;
    color: #222222;               /* 深灰近黑文字 */
    background-color: transparent;
}
</string>
       </property>
       <property name="text">
        <string>   ACCOUNT : </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Password">
       <property name="styleSheet">
        <string notr="true">QLabel {
    font: 13pt &quot;Microsoft YaHei UI&quot;;
    color: #222222;               /* 深灰近黑文字 */
    background-color: transparent;
}
</string>
       </property>
       <property name="text">
        <string> PASSWORD : </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="AccountEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    font: 12pt &quot;Microsoft YaHei UI&quot;;
    border: 2px solid #888888;
    border-radius: 6px;
    padding: 4px;
    background-color: #f9f9f9;
    color: #111111;
}
</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="PasswordEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    font: 12pt &quot;Microsoft YaHei UI&quot;;
    border: 2px solid #888888;
    border-radius: 6px;
    padding: 4px;
    background-color: #f9f9f9;
    color: #111111;
}
</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="LoginButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    font: 13pt &quot;Microsoft YaHei UI&quot;;
    border: 3px solid black;
    border-radius: 10px;
    background-color: #dcdcdc;  /* 淺灰色背景 */
    color: #111111;             /* 深黑色文字 */
    padding: 6px;
}
</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="Admin_check">
       <property name="cursor">
        <cursorShape>CrossCursor</cursorShape>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QCheckBox {
    font: 8pt &quot;Microsoft YaHei UI&quot;;
    color: #222222;  /* 深灰色文字 */
}

QCheckBox::indicator {
    background-color: #f0f0f0;
    border: 2px solid #888888;
    width: 10px;
    height: 10px;
}

QCheckBox::indicator:checked {
    background-color: #4CAF50;  /* 勾選後顯示綠色 */
    border: 1px solid #4CAF50;
}
</string>
       </property>
       <property name="text">
        <string>Admin</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
